### PR TITLE
Don't do r2r-jitstress jobs for non Checked configurations.

### PR DESCRIPTION
**Pull request by LLITCHEV**

### CODE REVIEW

**Comment by LLITCHEV** [comment]
@dotnet-bot test ci please

**Comment by LLITCHEV** [comment]
@dotnet-bot test ci please

**Comment by LLITCHEV** [comment]
@mmitche @RussKeldorph @ramarag @wtgodbe PTAL

**Comment by LLITCHEV** [comment]
@dotnet-bot test ci please

File: netci.groovy [hunk 1]
```diff
@@ -1264,6 +1264,12 @@ combinedScenarios.each { scenario ->
                         }
                     }
                     else {
+                        // If this is a r2r jitstress, jitstressregs, jitminopts or forcerelocs scenario
+                        // and configuration is not Checked, bail out.
+                        if (configuration != 'Checked' && Constants.r2rJitStressScenarios.indexOf(scenario) != -1) {
```
Comment (RussKeldorph) [inline netci.groovy]: Should you be indexing into the array with scenario

**Comment by RussKeldorph** [comment]
LGTM

**Comment by mmitche** [comment]
LGTM

**Comment by wtgodbe** [comment]
LGTM

**Pull request merged**